crates/biome_css_semantic/src/events.rs

```diff
@@ -103,15 +98,11 @@ impl SemanticEventExtractor {
                 if let Some(property_name) = node.first_child().and_then(|p| p.first_child()) {
                     if let Some(value) = property_name.next_sibling() {
                         self.stash.push_back(SemanticEvent::PropertyDeclaration {
+                            node: node.clone(),
                             property: CssProperty {
-                                name: property_name.text_trimmed().to_string(),
-                                range: property_name.text_trimmed_range(),
-                            },
-                            value: CssValue {
-                                text: value.text_trimmed().to_string(),
-                                range: value.text_trimmed_range(),
+                                node: property_name,
                             },
-                            range: node.text_range(),
+                            value: CssValue { node: value },
```

Comment:
imho: can be change to `property_value` as it aligns well with `property_name`?

```suggestion
                            value: CssValue { node: property_value },
```